cargo-shuttle/src/client.rs

```diff
@@ -32,7 +32,7 @@ impl Client {
     }
 
     pub async fn deploy(&self, data: Vec<u8>, project: &ProjectName) -> Result<String> {
-        let path = format!("/deploy/{}", project.as_str(),);
+        let path = format!("/project/{}", project.as_str(),);
```

Comment:
I was fine with the new name. Maybe should have an API route discussing with the next cycle plan